scripts/run-instrumented-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo =============================================
+echo Starting ADB Daemon
+echo =============================================
+adb start-server
+emulator @test -no-window -no-audio -wipe-data &
+sleep 30
```

Comment:
sleep 30 the best we can do?  Maybe something here: https://stackoverflow.com/questions/3634041/detect-when-android-emulator-is-fully-booted or here https://androidstudio.googleblog.com/2019/05/emulator-2907-canary.html ?

maybe -no-boot-anim ?

Reply:
It's a headless emulator.  adb wait-for-device was hanging so i added the sleep for now while investigating other alternatives.   
 It's not actually necessary... more a reminder to future engineers....The emulator currently starts in 1-1.5 minutes and gradle is currently looking for depedencies for at least 4 minutes.

Reply:
Cool.  I do like scripting that says "Sleeping for <x> <timeunits> because <reason>" so that operators reading the term know what's happening.  I'd rather have it be bulletproof so that we can't break it by fixing the build speed (in some better, future world)... what happens if it outruns the emulator, though?